### DISCUSSION TITLE
父节点不存在的解决方案

### DISCUSSION THREAD

**Discussion by 265525** [Mufanc/iSmartAuto#46] [Ideas]
![B%ZLJ8@`(AW13IT1J(BQ2](https://user-images.githubusercontent.com/30653455/219827186-4b766380-7e04-4597-b733-c874958b7215.png)
我没有详细研究过这个软件的具体逻辑，但我分析应该都是一个流程。此方法可能有风险，请慎重选择，我去掉了作者的检测节点机制的判断逻辑。就可以工作了。

**Comment by Aaphyvia** [comment]
改了之后脚本直接不可用了是为啥
